### DISCUSSION TITLE
Shadow Element for user-agent.

### DISCUSSION THREAD

**Discussion by Lord-Shad0w** [webdriverio/webdriverio#13934] [Q&A]
I have an element in the webpage. On inspecting it was not showing in dom then in chrome dev tools I enabled an option "Shadow root user-agent" after that the element is coming under shadow-root (user-agent). Now I am trying to acces that but its not happening. I even tried to acces it with dev console but that also working so is there any way to access such elements?. Is there any way to interact with user-agent shadow Elements? I tried $shadow() but it's now working. Normal shadow Elements (without user-agent) are working fine with $shadow
![IMG20241128202920](https://github.com/user-attachments/assets/d3943e05-654e-482b-9930-ed6bbbaa51f6)
![IMG20241128202849](https://github.com/user-attachments/assets/5477704a-dda9-4101-931d-22f4716c9c4a)
![IMG20241128202833](https://github.com/user-attachments/assets/bb959f85-aa6b-41c4-8a58-50e0d8d66330)
![IMG20241128202751](https://github.com/user-attachments/assets/7d66de4b-7812-49ef-9587-9e4e28f19e07)

**Comment by christian-bromann** [comment]
@Lord-Shad0w you are working with a native web component, which I don't think you have access to their ShadowRoot. You can get or set the value for a range element like any other input, there should be no need for you to access the inner elements.
